bee-bundle/src/transaction.rs

```diff
@@ -7,7 +7,7 @@ use std::fmt;
 use std::iter;
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub struct Payload(TritBuf<T1B1Buf>);
+pub struct Payload(pub TritBuf<T1B1Buf>);
```

Comment:
Can you make the inner fields `pub(crate)` ?

Reply:
yes, nice